config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get 'users/:id', to: 'users#show'
+  get 'users/:id/posts', to: 'posts#index'
+  get 'users/:user_id/posts/:post_id', to: 'posts#show'
+  root "users#index"
```

Comment:
- [ ] I suggest that you use `resource routing` because it allows you to quickly declare all of the common routes for a given resourceful controller
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Implemented